openedx/core/lib/api/view_utils.py

```diff
@@ -118,3 +121,57 @@ def _decorator(func_or_class):
             func_or_class.permission_classes += (IsUserInUrl,)
         return func_or_class
     return _decorator
+
+
+def add_serializer_errors(serializer, data, field_errors):
+    """
+    Adds errors from serializer validation to field_errors. data is the original data to deserialize.
+    """
+    if not serializer.is_valid():  # pylint: disable=maybe-no-member
+        errors = serializer.errors  # pylint: disable=maybe-no-member
+        for key, error in errors.iteritems():
+            field_errors[key] = {
+                'developer_message': u"Value '{field_value}' is not valid for field '{field_name}': {error}".format(
+                    field_value=data[key] if key in data else '', field_name=key, error=error
+                ),
+                'user_message': _(u"This value is invalid."),
+            }
+
+
+class RetrievePatchAPIView(RetrieveModelMixin, UpdateModelMixin, GenericAPIView):
```

Comment:
Yes. add_serializer_errors will be run completely in the test where the team name is too long, for example.